src/bcf/header.rs

```diff
@@ -481,6 +491,10 @@ quick_error! {
             description("unknown sequence")
             display("sequence {} not found in header", name)
         }
+        UnknownRID(rid: u32) {
```

Comment:
Can you make this a separate error? The reason is that neither rid2name nor name2rid can throw both errors.

Reply:
Yes, that makes sense. It's now has a dedicated `RidIndexError::UnknownIndex` error.